docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst

```diff
@@ -45,13 +45,11 @@ Stomach Cancer
 +=======+============================+
 | MST   | Mean Sojourn Time          |
 +-------+----------------------------+
-| AST   | Average Survival Time      |
-+-------+----------------------------+
 | PC    | Pre-Clinical Cancer        |
 +-------+----------------------------+
-| S     | Susceptible                |
+| S     | Susceptible in sim pop     |
 +-------+----------------------------+
-| C     | With Cancer condition      |
+| C     | Clinical cancer in sim pop |
 +-------+----------------------------+
 
```

Comment:
Can you add definition of `sim pop`, such as general population - (clinical stomach cancer cases +  pre-clinical stomach cancer cases detected through baseline screening).

Reply:
or just strike out the phrase "in sim pop" --- I think it is clear enough without it